master/internal/model/authz_iface.go

```diff
@@ -34,6 +34,9 @@ type ModelAuthZ interface {
 	CanCreateModel(ctx context.Context,
 		curUser model.User, workspaceID int32,
 	) error
+	// POST /api/v1/models/{model_name}/move
+	CanMoveModel(ctx context.Context, curUser model.User, model *modelv1.Model, fromWorkspaceID,
```

Comment:
`fromWorkspaceID` should also be `int32` typed. Even if go allows this for short, I'd rather we were clearer about it here and in authz_basic_impl